src/lib/modules/Base/index.js

```diff
@@ -0,0 +1,28 @@
+class BaseModule {
```

Comment:
It's not the responsability of a module to dispatch a message through sockets.

Let's take a step back: what a module is supposed to do in this context ? Let's say, what `UIViewManager` is suppose to do ? I'm not sure it's to send messages through sockets 😝